2020-06-11 17:51:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE NEWS(ID BIGINT PRIMARY KEY AUTO_INCREMENT,
TITLE TEXT,
CONTENT TEXT,
URL VARCHAR(100),
CREAT_AT TIMESTAMP,
MODIFIED_AT TIMESTAMP)
MODIFIED_AT[*] TIMESTAMP) "; SQL statement:
create table News(id bigint primary key auto_increment,
title text,
content text,
url varchar(100),
creat_at timestamp,
modified_at timestamp)
modified_at timestamp) [42000-199]
2020-06-11 17:54:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSEDS" not found; SQL statement:
select link from links_already_processedS [42102-199]
2020-06-11 17:56:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSEDS" not found; SQL statement:
select link from links_already_processedS [42102-199]
2020-06-12 13:44:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE TABLE[*] LINKS_ALREADY_PROCESSED "; expected "identifier"; SQL statement:
delete table LINKS_ALREADY_PROCESSED [42001-199]
2020-06-12 13:44:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELATE[*] TABLE LINKS_ALREADY_PROCESSED "; expected "DELETE, DROP, DECLARE, DEALLOCATE"; SQL statement:
delate table LINKS_ALREADY_PROCESSED [42001-199]
2020-06-12 13:44:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE TABLE[*] LINKS_ALREADY_PROCESSED "; expected "identifier"; SQL statement:
delete table LINKS_ALREADY_PROCESSED [42001-199]
2020-06-12 13:44:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE TABLE[*] FROM LINKS_ALREADY_PROCESSED "; expected "identifier"; SQL statement:
delete table from LINKS_ALREADY_PROCESSED [42001-199]
2020-06-12 13:45:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSED" already exists; SQL statement:
create table LINKS_ALREADY_PROCESSED (link varchar (1000)) [42101-199]
2020-06-12 13:46:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELATE[*] FROM  LINKS_TO_BE_PROCESSED WHERE LINK = ?  "; expected "DELETE, DROP, DECLARE, DEALLOCATE"; SQL statement:
DELATE FROM  LINKS_TO_BE_PROCESSED where link = ?  [42001-199]
2020-06-12 13:47:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_TO_BE_PROSESSED" not found; SQL statement:
INSERT INTO LINKS_TO_BE_PROSESSED(link) values (?) [42102-199]
2020-06-12 13:47:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_TO_BE_PROSESSED" not found; SQL statement:
INSERT INTO LINKS_TO_BE_PROSESSED(link) values (?) [42102-199]
2020-06-12 13:52:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:72)
2020-06-12 14:00:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:65)
2020-06-12 14:04:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:61)
2020-06-12 14:25:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Method is only allowed for a query. Use execute or executeUpdate instead of executeQuery; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [90002-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.Prepared.query(Prepared.java:229)
	at org.h2.command.CommandContainer.query(CommandContainer.java:145)
	at org.h2.command.Command.executeQuery(Command.java:202)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:115)
	at com.github.Hana.Main.main(Main.java:63)
2020-06-12 14:27:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(100)": "'https://cj.sina.cn/article/norm_detail?url=https%3A%2F%2Ffinance.sina.cn%2F2020-06-12%2Fdetail-iirczymk6592669.d.html&vt=4&pos=... (130)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 14:31:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:61)
2020-06-12 14:36:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:36:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:36:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:36:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:37:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:37:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:37:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:37:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:37:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:37:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSE" not found; SQL statement:
INSERT INTO LINKS_ALREADY_PROCESSE(link) values (?) [42102-199]
2020-06-12 14:38:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS_ALREADY_PROCESSEED" not found; SQL statement:
select * from LINKS_ALREADY_PROCESSEED [42102-199]
2020-06-12 14:39:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINK" not found; SQL statement:
delete link from LINKS_ALREADY_PROCESSED [42102-199]
2020-06-12 14:39:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "LINKS" not found; SQL statement:
delete links from LINKS_ALREADY_PROCESSED [42102-199]
2020-06-12 14:40:18 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.Hana.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-12 14:42:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ELECT[*] * FROM LINKS_TO_BE_PROCESSED "; expected "EXPLAIN, EXECUTE"; SQL statement:
elect * from LINKS_TO_BE_PROCESSED [42001-199]
2020-06-12 14:51:39 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.Hana.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-12 15:25:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.Hana.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-12 15:32:32 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at com.github.Hana.Main.main(Main.java:25)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/jinhua/IdeaProjects/news-crewler/news.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 19 more
2020-06-12 16:01:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iirczymk6537288.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:01:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iircuyvi8008044.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:01:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fwm%2F2020-06-12%2Fdetail-iircuyvi8007389.d.html%3Fvt%3D4%26pos%3D1... (2061)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:01:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fwm%2F2020-06-12%2Fdetail-iirczymk6536333.d.html%3Fvt%3D4%26pos%3D1... (2061)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:01:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iirczymk6536334.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iirczymk6534771.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:66)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:66)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:66)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:66)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:66)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:66)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.disposeUsedLinkFromDatabase(Main.java:95)
	at com.github.Hana.Main.main(Main.java:76)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:78)
2020-06-12 16:01:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1547)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:347)
	at com.github.Hana.Main.main(Main.java:49)
2020-06-12 16:05:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fpi%2Fpf%2F2014-11-24%2Fdetail-icesifvw8291824.d.html%3Fvt%3D4&titl... (2054)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:26:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iirczymk6537288.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:26:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iircuyvi8008044.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:26:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fwm%2F2020-06-12%2Fdetail-iircuyvi8007389.d.html%3Fvt%3D4%26pos%3D1... (2061)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:26:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fwm%2F2020-06-12%2Fdetail-iirczymk6536333.d.html%3Fvt%3D4%26pos%3D1... (2061)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:26:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iirczymk6536334.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:26:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000)": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fgn%2Fvi%2F2020-06-12%2Fdetail-iirczymk6534771.d.html%3Fvt%3D4%26po... (2066)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
2020-06-12 16:27:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column """LINK"" VARCHAR(1000) SELECTIVITY 20": "'http://share.sina.cn/callback?vt=4&url=%2F%2Fgames.sina.cn%2Fpi%2Fpf%2F2014-11-24%2Fdetail-icesifvw8291824.d.html%3Fvt%3D4&titl... (2054)"; SQL statement:
INSERT INTO LINKS_TO_BE_PROCESSED(link) values (?) [22001-199]
